themes-default/slim/src/components/show.vue

```diff
@@ -70,11 +73,12 @@ export default {
                 return defaults.show;
             }
 
+            // With the current mako there is no reason to enable this IMO.
             // Not detailed
-            if (!show.seasons) {
-                $store.dispatch('getShow', { id, indexer, detailed: true });
-                return getShowById({ indexer, id });
-            }
+            // if (!show.seasons) {
+            //     $store.dispatch('getShow', { id, indexer, detailed: false });
+            //     return getShowById({ indexer, id });
+            // }
```

Comment:
It is used by the "jump to season"
https://github.com/pymedusa/Medusa/blob/918cfe7ce01808499585ffe6c1356fd5ed369f76/themes-default/slim/views/partials/showheader.mako#L32-L56

Reply:
Tnx allot! I'll look into it.